sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -9,10 +9,8 @@
   `_OTELRESOURCE_` metrics.
   ([#41761](https://github.com/Azure/azure-sdk-for-net/pull/41761))
 
-* Fixed an issue where tags associated with Exceptions, specifically those logged using
-  [Activity.RecordException()](https://github.com/open-telemetry/opentelemetry-dotnet/blob/f69c025178216d1879e5884f0a74ab15ca759a23/src/OpenTelemetry.Api/Trace/ActivityExtensions.cs#L81),
-  were not being included.
-  Now, tags linked to an `ActivityEvent` of type exception are correctly exported as Custom Properties.
+* Fixed an issue where tags associated with Exceptions were not being included.
+  Now, tags linked to an `ActivityEvent` following the otel convention for storing exception are correctly exported as Custom Properties.
```

Comment:
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-spans.md add this link too.